jupyterhub/alembic/env.py

```diff
@@ -12,9 +12,16 @@
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
 if 'jupyterhub' in sys.modules:
+    from traitlets.config import MultipleInstanceError
     from jupyterhub.app import JupyterHub
+    app = None
     if JupyterHub.initialized():
-        app = JupyterHub.instance()
+        try:
+            app = JupyterHub.instance()
+        except MultipleInstanceError:
+            # could have been another Application
```

Comment:
It's not entirely clear to me what other Application had an instance. Another JupyterHub running for a different purpose or group of users?

Reply:
Where I ran into this, the Application was IPython.